templates/docker-compose.yml

```diff
@@ -7,18 +7,22 @@ x-logging: &default-logging
     max-file: "4"
 
 networks:
-  # communication to web and clients
-  lemmyexternalproxy:
+  default:
   # communication between lemmy services
   lemmyinternal:
     driver: bridge
-    internal: true
+  # communication to web and clients
+  lemmyexternalproxy:
+    driver: bridge
 services:
   proxy:
     image: nginx:1-alpine
+    hostname: proxy
     networks:
-      - lemmyinternal
-      - lemmyexternalproxy
+      lemmyinternal:
+        priority: 10
+      lemmyexternalproxy:
+        priority: 1000
```

Comment:
An explanation on the priority setting. Is there a reason why you chose 1000 instead of 100, or 20?

Reply:
It was a typo, meant to be "100." From a functional/technical standpoint, there's no detrimental effect having it that high or having that sort of delta between them. I'll adjust it down in the PR edit.